e2e/pages/Payments/PaymentsPage.ts

```diff
@@ -175,6 +177,82 @@ class PaymentsPage {
     });
     await expect(this.paymentRetryButton).toBeHidden();
   }
-}
 
+  async verifyPaymentOptionUnderPaymentData({
+    paymentNumber = 1,
+  }: {
+    paymentNumber?: number;
+  }) {
+    await this.page.waitForLoadState('networkidle');
+    await this.paymentDropdown.click();
+    const contents = await this.paymentDropdown.allTextContents();
+    const currentPaymentLabel = `Payment #${paymentNumber}  Closed`;
+    await this.verifyLabelExist(currentPaymentLabel, contents);
+    const nextPaymentLabel = `Payment #${paymentNumber + 1}  Open`;
+    await this.verifyLabelExist(nextPaymentLabel, contents);
```

Comment:
It may be clearer to place variables on top of the function as in:

async verifyPaymentOptionUnderPaymentData({
    paymentNumber = 1,
  }: {
    paymentNumber?: number;
  }) {
    const contents = await this.paymentDropdown.allTextContents();
    const currentPaymentLabel = `Payment #${paymentNumber}  Closed`;

And then rest of the code

Reply:
I will move currentPaymentLabel and nextPaymentLabel on the top of the function. 
The contents needs to put under the await this.paymentDropdown.click(); to get the options